CHANGELOG.md

```diff
@@ -3,22 +3,23 @@
 ## unreleased
 
 ### Improvements
-+ Add workflow for examples
-+ New package `Web`, It can be used in Dapps to interact with ASAs and Apps
-    + Compatible with `algosigner`.
-    + Support `executeTransaction` function with web package
-+ Move Error lists, BuilderError, mkTransaction to web package
-+ Reuse mkTransaction, Errors in algob and runtime
++ Added github workflows/examples.yaml to execute [`/examples`](https://github.com/scale-it/algo-builder/tree/master/examples) on a private net, on push.
```

Comment:
what does it mean "on push"? Could you reword please?

Reply:
ok